presto-testing-docker/src/main/java/io/prestosql/testing/docker/DockerContainer.java

```diff
@@ -63,20 +63,20 @@
 
     private Map<Integer, Integer> hostPorts;
 
-    public DockerContainer(String image, List<Integer> ports, Map<String, String> environment, CheckedConsumer<HostPortProvider> healthCheck)
+    public DockerContainer(String image, List<Integer> ports, Map<String, String> environment, List<String> capabilities, CheckedConsumer<HostPortProvider> healthCheck)
     {
         this.image = requireNonNull(image, "image is null");
         this.environment = ImmutableMap.copyOf(requireNonNull(environment, "environment is null"));
         try {
-            startContainer(ports, healthCheck);
+            startContainer(ports, healthCheck, capabilities);
```

Comment:
Assign it to a field instead of passing around.
Also validate it's not null and doesn't contain nulls (`ImmutableList.copyOf`), instead of relying on docker lib's `.capAdd` doing the validation.